### ISSUE TITLE
Enhancement: scroll to zoom

### ISSUE THREAD

**Issue by zanthius** [treyp/thebutton#2]
Hi, Just wondering if it's possible to have a zoom control on the graph?

Thanks.

**Comment by treyp** [comment]
yeah, i've wanted this myself. i've added it to [the to do list](https://github.com/treyp/thebutton/blob/master/README.md#to-do-list).
